datadog/plugins/mysql.sls

```diff
@@ -0,0 +1,13 @@
+{% set dd_mysql = salt.pillar.get(
+    'datadog:integrations:mysql',
+    {
+        'password': 'Ch@ng3me!',
+        'host': '*.*',
```

Comment:
Similar comment as mongo: Can entries for these be added to `pillar.example`, even if it has to be commented out by default?